### PR TITLE
docs: Updated GSX Integration Docs to reflect current implementation

### LINKED ISSUE #934: Update gsx fueling page to automatic fueling start.
The refuelling process with gsx got changed and should be adapted in the docs as well.

https://github.com/flybywiresim/aircraft/pull/8407

https://docs.flybywiresim.com/fbw-a32nx/feature-guides/gsxintegration/fuel/

### PR DESCRIPTION
**Pull request by JasonO99**
Fixes #934 

## Summary
This PR updates the GSX Integration Documentation to better reflect changes made in recent PR's, and some other tweaks for typos etc.

- Corrected typo at the start of the Overview page
- Changed the hero image on the Overview page to something better to reflect the nature of this area of docs 
- Updated images to show the EFB page for the Payload and Fuel doc pages
- Updated Refuelling page to reflect the changes made in https://github.com/flybywiresim/aircraft/pull/8407
- Removed the information that states we offer the GSX Chocks and Cones feature as enabled in the included GSX profile, as this is no longer the case as of https://github.com/flybywiresim/aircraft/pull/8405
- Updated the GSX Aircraft Profile window to reflect the current default configuration also updated in the above PR.

### Location
https://docs.flybywiresim.com/fbw-a32nx/feature-guides/gsxintegration/

Discord username (if different from GitHub): jason_91

### CODE REVIEW

File: docs/fbw-a32nx/feature-guides/gsxintegration/fuel.md [hunk 1]
```diff
@@ -18,13 +18,15 @@ This page explains, how to use the fueling "synchronization" as-well as any cave
...
+Much of the refuelling process between the A32NX and GSX is automated, however some input is required from the user to set the fuel and initiate refuelling.
 
 1. Fueling
-      - The "play" button is disabled until the fuel truck has arrived, and the hose is connected.
-      - Select `Request Fueling` from the GSX Menu.
-      - Wait for the fuel truck to arrive and connect the fuel hose.
-      - Set the [fuel quantity](../flypados3/ground.md#fuel-and-de-fuel) that you desire and press the "play" button.
-      - You can witness outside as the fuel truck begins to re-fuel your aircraft.
+      - To use GSX as part of the refuelling process, 'Refuel Time' should be `Real` or `Fast`. Selecting `Instant` will add the fuel immediately, and will not involve GSX.
+      - Ensure you have syncronised your required fuel with your Simbrief flight plan, or manually adjusted the slider to set how much fuel you require.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/gsxintegration/fuel.md]:
```suggestion
      - Ensure you have synchronised your required fuel with your Simbrief flight plan, or manually adjust the slider to set how much fuel you require.
```

**Review by Valastiri — COMMENTED** [review]
All is generally well :) As indicated in Discord let's look at the images before merging this. Appreciate the update!

File: docs/fbw-a32nx/feature-guides/gsxintegration/fuel.md [hunk 2]
```diff
@@ -18,13 +18,15 @@ This page explains, how to use the fueling "synchronization" as-well as any cave
 ## Usage
 ![Fuel Start](../../assets/gsxintegration/gsx-fuel-start.png)
 
-You may notice the fuel page is slightly different, with the Play button being greyed out. This is because for the sake of GSX flow, you cannot begin fueling until the fuel hose is connected.
+Much of the refuelling process between the A32NX and GSX is automated, however some input is required from the user to set the fuel and initiate refuelling.
```
Comment (Sleinmaster) [inline docs/fbw-a32nx/feature-guides/gsxintegration/fuel.md]:
```suggestion
Much of the refuelling process between the A32NX and GSX is automated; However some input is required from the user to set the fuel and initiate refuelling.
```

File: docs/fbw-a32nx/feature-guides/gsxintegration/fuel.md [hunk 3]
```diff
@@ -18,13 +18,15 @@ This page explains, how to use the fueling "synchronization" as-well as any cave
...
 
 1. Fueling
-      - The "play" button is disabled until the fuel truck has arrived, and the hose is connected.
-      - Select `Request Fueling` from the GSX Menu.
-      - Wait for the fuel truck to arrive and connect the fuel hose.
-      - Set the [fuel quantity](../flypados3/ground.md#fuel-and-de-fuel) that you desire and press the "play" button.
-      - You can witness outside as the fuel truck begins to re-fuel your aircraft.
+      - To use GSX as part of the refuelling process, 'Refuel Time' should be `Real` or `Fast`. Selecting `Instant` will add the fuel immediately, and will not involve GSX.
+      - Ensure you have synchronised your required fuel with your Simbrief flight plan, or manually adjust the slider to set how much fuel you require.
```
Comment (Sleinmaster) [inline docs/fbw-a32nx/feature-guides/gsxintegration/fuel.md]:
```suggestion
      - Ensure you have synchronised your required fuel with your SimBrief flight plan, or manually adjust the slider to set how much fuel you require.
```

**Review by Sleinmaster — CHANGES_REQUESTED** [review]

**Comment by Valastiri** [comment]
LGTM -- sleins review seems handled will merge thanks !

**Pull request merged**